common/app/model/content.scala

```diff
@@ -304,6 +306,17 @@ final case class Content(
   val nonCompliantOutbrainAmp = (hasStoryPackage && tags.series.nonEmpty) || (tags.series.length > 1)
 }
 
+object CardStylePicker {
```

Comment:
Is there a better place for this object to live at. This `Content` file is already huge and I am not sure this belongs in it?